src/components/UserMessageWithBodyInput.tsx

```diff
@@ -72,9 +69,9 @@ export default function UserMessageWithBodyInput(props: Props) {
   return (
     <Root>
       {displayProfileImage ? (
-        <ImageContainer>
+        <div>
```

Comment:
do we even need this?

Reply:
I'm not sure; I just removed the styled components since it was an empty ` div`` `